docs/proxy/residential_proxy/tips_and_tricks.md

```diff
@@ -5,29 +5,29 @@ paths:
 
-## The traffic used by automated browsers needs to be controlled
+## [](#control-traffic-used-by-automated-browsers) Control traffic used by automated browsers
 
-Since Residential proxy is priced by data traffic used, it's very easy to realy quickly use up all prepaid traffic, especialy when accessing websites with large files loaded on every page.
+Residential proxy is priced by data traffic used. Thus, it's easy to quickly use up all your prepaid traffic. In particular, when accessing websites with large files loaded on every page.
 
-There are two main ways to reduce the traffic use of your solutions:
+There are two main ways to reduce your traffic use.
 
 *   [Block requests which are not required for your solution](https://help.apify.com/en/articles/2423246-block-requests-in-puppeteer)
 *   [Cache responses that you require but do not wish to load again and again](https://help.apify.com/en/articles/2424032-cache-responses-in-puppeteerr)
 
-When both of these options are combined, the data traffic of your solution can be reduced dramatically.
+Combined, these options can significantly reduce your solution's data traffic.
 
-## The speed of connected proxy can vary greatly
+## [](#connected-proxy-speed-variation) Connected proxy speed variation
 
-Each host on the Residential proxy network uses a different device, has different network speed and different latencies. This means that request made with one session can be extremely fast, while another request with different session can be extremely slow. The difference can be from a few milliseconds to a few seconds.
+Each host on the residential proxy network uses a different device. They have different network speeds and different latencies. This means that requests made with one session can be extremely fast, while another request with a different session can be extremely slow. The difference can range from a few milliseconds to a few seconds.
 
-If your solution requires quickly loaded content, the best option is to set session, try a small request and see if the response time is acceptable. If it is, you can use this session for other requests. Otherwise, repeat the attempt with a different session.
-Just remember that sessions are not persistent, so from time to time, you will have to reevaluate the speed of the session and switch to a different one if needed.
+If your solution requires quickly loaded content, the best option is to set a session, try a small request and see if the response time is acceptable. If it is, you can use this session for other requests. Otherwise, repeat the attempt with a different session.
 
-## Connections can be interrupted
+Just remember that sessions are not persistent, so from time to time, you will have to re-evaluate the session's speed and switch to a different one.
```

Comment:
I think a better explanation would be that the sessions are persistent, but they can be destroyed at any time, because the host devices can get turned off or disconnected.